test-specifications/crypto-service/accountDeleteTransaction.md

```diff
@@ -18,23 +18,112 @@ https://github.com/hashgraph/hedera-protobufs/blob/main/services/crypto_delete.p
 
 https://github.com/hashgraph/hedera-protobufs/blob/main/services/response_code.proto
 
-## Initialisation:
+**Mirror Node APIs:**
 
-```jsx
-new AccountDeleteTransaction()
+https://docs.hedera.com/hedera/sdks-and-apis/rest-api
 
-//example
+## JSON-RPC API Endpoint Documentation
 
-const transaction = new AccountDeleteTransaction()
-  .setAccountId(accountId)
+### Method Name
+
+`deleteAccount`
+
+### Input Parameters
+
+| Parameter Name    | Type   | Required/Optional | Description/Notes                                              |
+|-------------------|--------|-------------------|----------------------------------------------------------------|
+| deleteAccountId   | string | optional          | The ID of the account to delete.                               |
+| transferAccountId | string | optional          | The ID of the account to which to transfer remaining balances. |
+
+### Output Parameters
+
+| Parameter Name | Type   | Description/Notes                                                                     |
+|----------------|--------|---------------------------------------------------------------------------------------|
+| status         | string | The status of the submitted `AccountDeleteTransaction` (from a `TransactionReceipt`). |
+
+### Additional Notes
+
+The tests contained in this specification will assume that a valid account was already successfully created. Assume that the account was created with default values, unless specified otherwise. Any `<CREATED_ACCOUNT_ID>` tag will be the account ID of this created account. Any `<PRIVATE_KEY_OF_CREATED_ACCOUNT>` is the DER-encoded hex string of the private key of the account.
+
+## Property Tests
+
+### **Delete Account ID:**
```

Comment:
I suggest to add more negative cases:
- delete some admin account
- delete not existing account
- provide wrong private key
- etc.

Reply:
Added some more test cases